src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public  Random random = new Random();
+    public ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider making 'random' and 'colorSupplier' private and final if they do not need to be changed after the initialization of the Lottery instance. This will help to avoid accidental changes to these fields from within the class and make the class more thread-safe.